ui/css/pods-admin.css

```diff
@@ -351,13 +357,13 @@ div.pods-manage-field .pods-field-option-group .pods-field-option-group-label {
     border-bottom: none;
 }
 
-.pods-manage-field .pods-field-option .pods-form-ui-field-type-file ul.pods-flex-list li.pods-flex-name {
+/*.pods-manage-field .pods-field-option .pods-form-ui-field-type-file ul.pods-flex-list li.pods-flex-name {
```

Comment:
@sc0ttkclark Please confirm that I'm not forgetting an other usage of this code other than edit modals, settings and pods admin pages.
https://github.com/pods-framework/pods/commit/b553968d4b6b7f561fee08b5c1590f492891ad0f